From 7aa7701ba8faa3bb6f4131ebbed2f8ccfbe35f52 Mon Sep 17 00:00:00 2001
From: Quentin Perret <qperret@google.com>
Date: Wed, 5 Aug 2020 14:34:10 +0100
Subject: [PATCH 731/830] ANDROID: gki_defconfig: Enable MEDIA_SUPPORT_FILTER
X-NVConfidentiality: public

Since commit c6774ee035dc ("media: Kconfig: make filtering devices
optional"), a lot of options default to =y or =m if MEDIA_SUPPORT_FILTER
is not set. For instance, the current gki_defconfig builds a lot of
media tuner drivers as modules, without explicitly mentioning them. As
such, all these risk to be mistaken as GKI modules, even though we really
don't care.

Enable MEDIA_SUPPORT_FILTER to revert to the defaults we had in previous
android kernels.

Bug: 158655966
Signed-off-by: Quentin Perret <qperret@google.com>
Change-Id: I0f4ae305f43f88736d0a94bdf006f33aad7cf5c4
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 325008e65a51..7acb12bd49f2 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -319,6 +319,7 @@ CONFIG_MFD_ACT8945A=y
 CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_MEDIA_SUPPORT_FILTER=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index b3a2e9198b93..4bbfc6abbd93 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -281,6 +281,7 @@ CONFIG_WATCHDOG_CORE=y
 CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_MEDIA_SUPPORT_FILTER=y
 CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-- 
2.17.1

